FrontEndLib/Widget.h

```diff
@@ -276,6 +276,7 @@ class CWidget : public CAttachableObject
 	virtual bool         Load();
 	bool        LoadChildren();
 	void        PaintChildren(bool bUpdateRects = false);
+	void        PreventBubbling();
```

Comment:
Suggest rename to clarify context: PreventEventBubbling or PreventEventBubblingUp

Also, suggest adding a parameter `bool bFlag = true`, so this call may also be made to re-enable bubbling by providing a `false` argument.